tools/go.mod

```diff
@@ -0,0 +1,63 @@
+module tools
+
+go 1.20
```

Comment:
It might make sense to keep this in sync with the root `go.mod` go directive more for consistency and "update this everywhere" rather than it actually affecting consumers of the Go module itself 👍